packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -19,13 +19,7 @@ import BrowserDetection from '../common/browser-detection';
 
 const {isBrowser} = BrowserDetection();
 
-// TODO: in internal-media-core add appropriate types in MultistreamConnectionConfig for enable main audio and video
-type MultistreamConnectionConfig = ConstructorParameters<
-  typeof MultistreamRoapMediaConnection
->[0] & {
-  enableMainAudio: boolean;
-  enableMainVideo: boolean;
-};
+type MultistreamConnectionConfig = ConstructorParameters<typeof MultistreamRoapMediaConnection>[0];
```

Comment:
ideally we should just export MultistreamConnectionConfig from internal-media-core so that we wouldn't have to do this hack here

Reply:
I agree. Have added a Todo along with a Jira to track it. Will update.